components/engine/src/rocks/util/metrics_flusher.rs

```diff
@@ -81,6 +81,7 @@ fn flush_metrics(db: &DB, name: &str, shared_block_cache: bool) {
         }
     }
     flush_engine_properties(db, name, shared_block_cache);
+    flush_engine_iostall_properties(db, name);
```

Comment:
it's unnecessary to get iostall everytime, we can just do it when `on_stall_conditions_changed`. And maybe you can use existing metrics `STORE_ENGINE_WRITE_STALL_VEC` instead of creating a new one?

Reply:
No, I thought it was for another metric. If it hasn't been used till now, I would  change it.

Reply:
It does have been used, but I think these two metrics can be merged into one for conveniency.

Reply:
oh, sorry. I miswrited the metrics, it should be `STORE_ENGINE_STALL_CONDITIONS_CHANGED_VEC`

Reply:
But there is some difference. All property of `MapProperties` were cumulative value. So I must use prometheus expression to calculate it's value at every moment, such as `sum(increase(tikv_engine_write_stall_reason{instance=~"$instance", db="$db"}[1m])) by (type)`

Reply:
maybe we can deprecate STORE_ENGINE_STALL_CONDITIONS_CHANGED_VEC? seem now it is redundant to have two similar metrics.

Reply:
Yes, I agree. "Stall conditions changed of each CF" just show stall condition of each cf, but it does not tell us what cause write stall.

Reply:
+1 for removing STORE_ENGINE_STALL_CONDITIONS_CHANGED_VEC